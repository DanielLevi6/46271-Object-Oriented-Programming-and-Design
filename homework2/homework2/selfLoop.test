# selfLoop test file

# Create a graph
CreateGraph myGraph

# Add a node
CreateNode n1 5
AddNode myGraph n1

# Add a self-loop
AddEdge myGraph n1 n1

# Print some stuff
ListNodes myGraph
ListChildren myGraph n1

# Find a path
FindPath myGraph n1 -> n1
