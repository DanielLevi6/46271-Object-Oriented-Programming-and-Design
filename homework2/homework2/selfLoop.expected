# selfLoop test file

# Create a graph
created graph myGraph

# Add a node
created node n1 with cost 5
added node n1 to myGraph

# Add a self-loop
added edge from n1 to n1 in myGraph

# Print some stuff
myGraph contains: n1
the children of n1 in myGraph are: n1

# Find a path
shortest path in myGraph: n1
